.github/workflows/release.yml

```diff
@@ -176,7 +176,7 @@ jobs:
           npm install
           cd ../tests
           npm install
-          npm run test;
+          npm run test -- -j $(($(nproc) / 2));
```

Comment:
A comment would be nice here. Is this detecting the number of cpus available and allocating half of them to the tests?